lib/liquid/expression.rb

```diff
@@ -45,5 +45,16 @@ def self.parse(markup)
         end
       end
     end
+
+    def self.format(expression)
+      case expression
+      when String
+        "'#{expression}'"
```

Comment:
This could lead to broken renders if the `expression` itself contains single quotes. For example:
```
{{input | substitute: '<subtext>', "two's company"}}
```
would get *formatted* into:
```
{{ input | substitute: '<subtext>', 'two's company' }}
```

Reply:
Yeah this is one that's going to need more tests, it can also go two ways I see. It can do detection and decide either single quotes or double quotes based on if the string contains the other or can standardise that liquid uses either single or double quotes by default for format and then make sure it can handle escaping in the strict parser.